.env.template

```diff
@@ -8,9 +8,7 @@ FLASK_DEBUG=true
 # Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
 SECRET_KEY=secret-key
 
-# Required to use any Trello API
-TRELLO_API_KEY=trello-api-key
-TRELLO_API_TOKEN=trello-api-token
-BOARD_ID=board-id
-TODO_LIST_ID=todo-list-id
-DONE_LIST_ID=done-list-id
\ No newline at end of file
+# Required to connect to MongoDB
+MONGOBD_PRIMARY_CONNECTION_STRING=mongodb-primary-connection-string
+MONGO_DATABASE_NAME=todo_app_database
+MONGO_COLLECTION_NAME=todo_app_collection
```

Comment:
Thanks for remembering to update this - made it very easy for me to test your code 😄